### PR TITLE
Submitting Review for Ticha by JessicaLSF

### PR DESCRIPTION
**Pull request by jessisanch12**

### CODE REVIEW

**Comment by jessisanch12** [comment]
Submitting Review for Ticha

File: 2018-10-25-Ticha-Review [hunk 1]
```diff
@@ -0,0 +1,50 @@
+Lillehaugen, Brook Danielle, George Aaron Broadwell, Michel R. Oudijk, & Laurie Allen. 2015.
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: Please format this article using Markdown. This includes using simple codes to italicize or bold titles, adding links, and adding a YAML header. The instructions are here:

http://www.halperta.com/criticalarchives/tutorial/writing-in-markdown/

File: 2018-10-25-Ticha-Review [hunk 2]
```diff
@@ -0,0 +1,50 @@
+Lillehaugen, Brook Danielle, George Aaron Broadwell, Michel R. Oudijk, & Laurie Allen. 2015.  
+*Ticha: a digital text explorer for Colonial Zapotec*.  
+First edition.(http://ticha.haverford.edu/)
+
+Ticha means language in Zapotec, an indigenous language spoken in Southern Mexico. Thicha is a digital platform
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: Typo: "thicha" in second sentence.

File: 2018-10-25-Ticha-Review [hunk 3]
```diff
@@ -0,0 +1,50 @@
+Lillehaugen, Brook Danielle, George Aaron Broadwell, Michel R. Oudijk, & Laurie Allen. 2015.  
+*Ticha: a digital text explorer for Colonial Zapotec*.  
+First edition.(http://ticha.haverford.edu/)
+
+Ticha means language in Zapotec, an indigenous language spoken in Southern Mexico. Thicha is a digital platform  
+that was and is being created by an interdisciplinary team of scholars from different fields in the United States  
+and Mexico at Haverford College Libraries: Brook D. Lillehaugen, George A. Broadwell, Michel R. Oudijk, and Laurie Allen.  
+Ticha analyzes data through two main technological platforms: FLEx which is Fieldworks Language Explorer, a system for
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: This sentence is a little confusing - it's unclear what the two platforms are. Perhaps break it into two sentences? I might use a different word in terms of platform. Perhaps "Ticha encodes the data in two ways:"

File: 2018-10-25-Ticha-Review [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+lexical and grammatical analysis, with Text Encoding Initiative standards for paleographic and translational representations  
+of texts as stated in About the Project. The corpus presented is from the colonial period which focus on three main works,  
+all of these texts come from the geographic region of Central Valley Zapotec Oaxaca. The first one is [Arte], a book published  
+in 1578 by Juan de Cordova, a Spanish priest in the Dominican order. The second is Doctrina christiana en lengua castellana y  
+çapoteca by Dominican priest Pedro de Feria. The third are the handwritten texts mainly written by native Zapotec speakers in  
+Zapotec and some in Spanish about religious issues, testaments, wills, deeds, and letters. The digital platform is organized  
+in the following three main categories: About, Colonial Zapotec, Explore the texts and Bibliography. The about presents  
+information about the project, the team, acknowledgments and a contact us. In addition to this information, at the bottom of  
+the about page, there are videos about the project with the team.
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: Think about reorganizing the information in this paragraph in terms of priority. I see three main descriptive points: the topic of the collection, the content of the collection, and the encoding system. I would organize them so the topic comes first, followed by content, and finally encoding.

File: 2018-10-25-Ticha-Review [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+
+The Colonial Zapotec section includes a linguistic background and cultural context to situate the project in contemporary
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: Please add links to these pages.

File: 2018-10-25-Ticha-Review [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+access and how it can and should be used as a political platform with active and dynamic elements (12). 
+
+To further address the question of accessibility, technology plays an important role in supporting a community. As Lara Putman  
+states, “the impact of digitization on the knowability of past processes of whatever scale and locale is significant” (380).  
+Ticha in this particular case, is serving a community that is actively working with the language, not only academics, but  
+the Zapotec community. The interactive, feasible, self-explanatory access Ticha provides is changing the ways archives  
+have been perceived to be. This is particularly seen with the implementation of audio recording from current community  
+members of the 600 words found in the dictionary, a feature that is still in progress. Through the work shown in Ticha,  
+one of their main goals is to extend bridges and render visible voices from the past, present and future to acknowledge  
+a community that has been silenced. Therefore, as the digital archive develops and expands, how will Ticha continue to
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: Either rewrite this final sentence as a statement, or end with a question mark.

**Review by halperta — CHANGES_REQUESTED** [review]
Jessica - Ticha is one of my favorite projects, and I'm really glad to see you reviewing it here. I really like how you address the question of access and community-centered digital humanities at the end. 

I've added some recommendations to improve the clarity of the prose, and requested that you reformat it using markdown (instructions below). Please submit your revisions by next Monday, October 29.

File: 2018-10-25-Ticha-Review [hunk 7]
```diff
@@ -1,50 +1,65 @@
...
-society and bring awareness to the users. Explore the texts has Arte en lengua zapoteca, which provides an outline, a PDF  
-version, a sample page that includes: original text, transcription of the text, modern Spanish and English, and an original  
-transcription of the text and a version in modern day Spanish. Doctrina christiana en lengua castellana has only information  
-about the priest, a PDF version of the text and a sample page with: the original text, a transcription and a linguistic analysis.  
-Finally, there are a total of fifty handwritten manuscripts, twenty one found in Spanish and twenty-nine in Zapotec, all are  
-organized by name, year, town, archive, type of document and language, original text, and a transcription. There is a sample  
-manuscript that includes the original page images in Zapotec and Spanish, transcriptions in Zapotec and in Spanish and following  
-metadata: archive name, collection, call number, pages, year, date, primary parties, witnesses, and scribe. 
+Ticha is an innovative and interactive contemporary digital platform that bring awareness to the users. Each of the
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: * brings

File: 2018-10-25-Ticha-Review [hunk 8]
```diff
@@ -1,50 +1,65 @@
...
-version, a sample page that includes: original text, transcription of the text, modern Spanish and English, and an original  
-transcription of the text and a version in modern day Spanish. Doctrina christiana en lengua castellana has only information  
-about the priest, a PDF version of the text and a sample page with: the original text, a transcription and a linguistic analysis.  
-Finally, there are a total of fifty handwritten manuscripts, twenty one found in Spanish and twenty-nine in Zapotec, all are  
-organized by name, year, town, archive, type of document and language, original text, and a transcription. There is a sample  
-manuscript that includes the original page images in Zapotec and Spanish, transcriptions in Zapotec and in Spanish and following  
-metadata: archive name, collection, call number, pages, year, date, primary parties, witnesses, and scribe. 
+Ticha is an innovative and interactive contemporary digital platform that bring awareness to the users. Each of the 
+sections contextualize the project and explore the texts section, provides a wide range of materials. For instance,
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: * providing

File: 2018-10-25-Ticha-Review [hunk 9]
```diff
@@ -1,50 +1,65 @@
...
-manuscript that includes the original page images in Zapotec and Spanish, transcriptions in Zapotec and in Spanish and following  
-metadata: archive name, collection, call number, pages, year, date, primary parties, witnesses, and scribe. 
+Ticha is an innovative and interactive contemporary digital platform that bring awareness to the users. Each of the 
+sections contextualize the project and explore the texts section, provides a wide range of materials. For instance, 
+Arte en lengua zapoteca, has an outline, a PDF version, of the text, and a sample page that includes: original text, 
+transcription of the text, modern Spanish and English, and the original transcription of the text and a version in 
+modern day Spanish. Doctrina christiana en lengua castellana includes a PDF version of the text and a sample page with: 
+the original text, a transcription and a linguistic analysis. Finally, there are a total of fifty handwritten manuscripts, 
+twenty one found in Spanish and twenty-nine in Zapotec, all are organized by name, year, town, archive, type of document
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: "all organized by"

File: 2018-10-25-Ticha-Review [hunk 10]
```diff
@@ -1,50 +1,65 @@
...
-metadata: archive name, collection, call number, pages, year, date, primary parties, witnesses, and scribe. 
+Ticha is an innovative and interactive contemporary digital platform that bring awareness to the users. Each of the 
+sections contextualize the project and explore the texts section, provides a wide range of materials. For instance, 
+Arte en lengua zapoteca, has an outline, a PDF version, of the text, and a sample page that includes: original text, 
+transcription of the text, modern Spanish and English, and the original transcription of the text and a version in 
+modern day Spanish. Doctrina christiana en lengua castellana includes a PDF version of the text and a sample page with: 
+the original text, a transcription and a linguistic analysis. Finally, there are a total of fifty handwritten manuscripts, 
+twenty one found in Spanish and twenty-nine in Zapotec, all are organized by name, year, town, archive, type of document 
+and language, original text, and a transcription. There is a sample manuscript that includes the original page images in 
+Zapotec and Spanish, transcriptions in Zapotec and in Spanish with the following metadata: archive name, collection, call
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: * Zapotec and Spanish, _as well as_ transcriptions in ...

File: 2018-10-25-Ticha-Review [hunk 11]
```diff
@@ -1,50 +1,65 @@
...
-for an academic audience in fields not limited to history, anthropology, linguistics, cultural studies and anyone who is interested  
-about colonial Zapotec. Ticha is also for the one and half million Zapotec speakers in Mexico and the United States, especially  
-in Los Angeles. In one of the videos, Janet Chávez Santiago, a Zapotec teacher from Teotitlan del Valle, Oaxaca stated the success  
-and engagement from her students when comparing modern Zapotec to colonial Zapotec.The interest from her students is vast especially  
-when many thought these corpuses of materials to be abroad, out of access. Scholars Carlos Aguirre and Javier Villa-Flores, state  
-how the archive legitimizes forms of authority and credibility; thus, in thinking with Ticha it is important to reflect on who has  
-access and how it can and should be used as a political platform with active and dynamic elements (12). 
+Ticha is primarily targeted for an academic audience in fields not limited to history, anthropology, linguistics, cultural 
+studies and anyone who is interested about colonial Zapotec. This platform was also created for the one and half million
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: * interested _in_ colonial Zapotec

File: 2018-10-25-Ticha-Review [hunk 12]
```diff
@@ -1,50 +1,65 @@
...
-Ticha in this particular case, is serving a community that is actively working with the language, not only academics, but  
-the Zapotec community. The interactive, feasible, self-explanatory access Ticha provides is changing the ways archives  
-have been perceived to be. This is particularly seen with the implementation of audio recording from current community  
-members of the 600 words found in the dictionary, a feature that is still in progress. Through the work shown in Ticha,  
-one of their main goals is to extend bridges and render visible voices from the past, present and future to acknowledge  
-a community that has been silenced. Therefore, as the digital archive develops and expands, how will Ticha continue to  
-serve different audiences, reflect several voices in the archive, and continue to emphasize a sense of reciprocity to  
-the Zapotec community. 
+To further address the question of accessibility, technology plays an important role in supporting a community. As Lara 
+Putman states, “the impact of digitization on the knowability of past processes of whatever scale and locale is significant”
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: * putnam

File: 2018-10-25-Ticha-Review [hunk 13]
```diff
@@ -1,50 +1,65 @@
...
+(380). Ticha in this particular case, is serving a community that is actively working with the language, not only academics, 
+but the Zapotec community. The interactive, feasible, self-explanatory access Ticha provides is changing the ways archives  
+have been perceived to be. This is particularly seen with the implementation of audio recording from current community members 
+of the 600 words found in the dictionary, a feature that is still in progress. Through the work shown in Ticha, one of their 
+main goals is to extend bridges and render visible voices from the past, present and future to acknowledge a community that 
+has been silenced. Therefore, as the digital archive develops and expands, how will Ticha continue to serve different audiences, 
+reflect several voices in the archive, and continue to emphasize a sense of reciprocity to the Zapotec community?
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: * reciprocity _with_ the Zapotec community.

File: 2018-10-25-Ticha-Review [hunk 14]
```diff
@@ -1,50 +1,65 @@
...
-çapoteca by Dominican priest Pedro de Feria. The third are the handwritten texts mainly written by native Zapotec speakers in  
-Zapotec and some in Spanish about religious issues, testaments, wills, deeds, and letters. The digital platform is organized  
-in the following three main categories: About, Colonial Zapotec, Explore the texts and Bibliography. The about presents  
-information about the project, the team, acknowledgments and a contact us. In addition to this information, at the bottom of  
-the about page, there are videos about the project with the team.
-
+Ticha means language in Zapotec, an indigenous language spoken in Southern Mexico. Ticha is a digital platform that 
+was and is being created by an interdisciplinary team of scholars from different fields in the United States and 
+Mexico at Haverford College Libraries: Brook D. Lillehaugen, George A. Broadwell, Michel R. Oudijk, and Laurie Allen.  
+The corpus presented is from the colonial period which focus on three main works, all of these texts come from the
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: from the colonial period _and focuses on_ three main works;

File: 2018-10-25-Ticha-Review [hunk 15]
```diff
@@ -1,50 +1,65 @@
...
-Zapotec and some in Spanish about religious issues, testaments, wills, deeds, and letters. The digital platform is organized  
-in the following three main categories: About, Colonial Zapotec, Explore the texts and Bibliography. The about presents  
-information about the project, the team, acknowledgments and a contact us. In addition to this information, at the bottom of  
-the about page, there are videos about the project with the team.
-
+Ticha means language in Zapotec, an indigenous language spoken in Southern Mexico. Ticha is a digital platform that 
+was and is being created by an interdisciplinary team of scholars from different fields in the United States and 
+Mexico at Haverford College Libraries: Brook D. Lillehaugen, George A. Broadwell, Michel R. Oudijk, and Laurie Allen.  
+The corpus presented is from the colonial period which focus on three main works, all of these texts come from the 
+geographic region of Central Valley Zapotec Oaxaca. The first one is [Arte], a book published in 1578 by Juan de
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: Please italicize the titles. You can do this by adding a * on each side of the word or phrase (with no spaces in between)

File: 2018-10-25-Ticha-Review [hunk 16]
```diff
@@ -1,50 +1,65 @@
...
-information about the project, the team, acknowledgments and a contact us. In addition to this information, at the bottom of  
-the about page, there are videos about the project with the team.
-
+Ticha means language in Zapotec, an indigenous language spoken in Southern Mexico. Ticha is a digital platform that 
+was and is being created by an interdisciplinary team of scholars from different fields in the United States and 
+Mexico at Haverford College Libraries: Brook D. Lillehaugen, George A. Broadwell, Michel R. Oudijk, and Laurie Allen.  
+The corpus presented is from the colonial period which focus on three main works, all of these texts come from the 
+geographic region of Central Valley Zapotec Oaxaca. The first one is [Arte], a book published in 1578 by Juan de 
+Cordova, a Spanish priest in the Dominican order. The second is Doctrina christiana en lengua castellana y çapoteca 
+by Dominican priest Pedro de Feria. The third set of meterials presented are handwritten texts mainly written by native
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: * materials

File: 2018-10-25-Ticha-Review [hunk 17]
```diff
@@ -1,50 +1,65 @@
...
+Zapotec speakers in Zapotec and some in Spanish about religious issues, testaments, wills, deeds, and letters. The digital 
+platform is organized in three main categories: About (https://ticha.haverford.edu/en/about/), Colonial Zapotec
+(https://ticha.haverford.edu/en/context/), Explore the texts (https://ds-omeka.haverford.edu/ticha/en/texts.html). In 
+addition to this information, at the bottom of the about page, one can find videos about the project with the team 
+(https://ticha.haverford.edu/en/about/). A opportunity that allows users to understand the motivation and processes of
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: * _This opportunity allows_ users to understand...

File: 2018-10-25-Ticha-Review [hunk 18]
```diff
@@ -1,50 +1,65 @@
...
+geographic region of Central Valley Zapotec Oaxaca. The first one is [Arte], a book published in 1578 by Juan de 
+Cordova, a Spanish priest in the Dominican order. The second is Doctrina christiana en lengua castellana y çapoteca 
+by Dominican priest Pedro de Feria. The third set of meterials presented are handwritten texts mainly written by native
+Zapotec speakers in Zapotec and some in Spanish about religious issues, testaments, wills, deeds, and letters. The digital 
+platform is organized in three main categories: About (https://ticha.haverford.edu/en/about/), Colonial Zapotec
+(https://ticha.haverford.edu/en/context/), Explore the texts (https://ds-omeka.haverford.edu/ticha/en/texts.html). In 
+addition to this information, at the bottom of the about page, one can find videos about the project with the team 
+(https://ticha.haverford.edu/en/about/). A opportunity that allows users to understand the motivation and processes of 
+the project. It is important to also note that the project has been lead by a team of linguists and anthropologists, 
+therefore its not surprising that Ticha encodes the data in the following two ways: FLEx which is Fieldworks Language
```
Comment (halperta) [inline 2018-10-25-Ticha-Review]: therefore _it's_ not surprising

**Review by halperta — CHANGES_REQUESTED** [review]
Jessica - this is so close to complete! There are just a few typos and small grammatical corrections left. I've marked them all, using italics to show recommended changes, or if it's a typo just writing the word in question. Once that's done, we can publish!

**Pull request merged**